enterprise/views.py

```diff
@@ -485,9 +518,9 @@ def create_enterprise_course_enrollment(request, consent_record, course_id):
             user_id=request.user.id
         )
         enterprise_customer_user.update_session(request)
-        __, created = EnterpriseCourseEnrollment.objects.get_or_create(
-            enterprise_customer_user=enterprise_customer_user,
-            course_id=course_id,
+        __, created = get_create_ent_enrollment(
+            course_id,
+            enterprise_customer_user
         )
         if created:
             track_enrollment('data-consent-page-enrollment', request.user.id, course_id, request.path)
```

Comment:
based on this, i don't think enterprise enrollments created through this route are guaranteed to come from enrollment urls, though i'm not sure what would result in an enterprise enrollment getting created here and not by something else...